src/style.css

```diff
@@ -0,0 +1,65 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+.search-form {
+  position: relative;
+  margin-top: 20px;
+}
+
+.image {
+  position: absolute;
```

Comment:
to avoid duplication create class image and apply to it position absolute, then create to additional classes image--big, image--small and apply other rules to them